acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/ibft2/ParameterizedBftTestBase.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.tests.acceptance.ibft2;
+
+import org.hyperledger.besu.tests.acceptance.dsl.AcceptanceTestBase;
+
+import java.util.Collection;
+
+import org.junit.Ignore;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+@RunWith(Parameterized.class)
+@Ignore("This is not a test class, it offers BFT parameterization only.")
+public class ParameterizedBftTestBase extends AcceptanceTestBase {
```

Comment:
Making this abstract so it's clear this isn't actually a test and class that just be used would be bit nicer than the ignore.

Reply:
done